app/docs/0.13.x/admin-api.md

```diff
@@ -425,6 +425,35 @@ HTTP 200 OK
 
 ---
 
+### Update Or Create Service
+
+#### Endpoint
+
+<div class="endpoint put">/services/{name or id}</div>
+
+Attributes | Description
+---:| ---
+`name or id`<br>**required** | The `id` **or** the `name` attribute of the Service to update.
+
+#### Request Body
+
+{{ page.service_body }}
+
+If a Service with the given name or id exists, the request updates it with the payload.
+If it doesn't exist, a new one will be created using the payload and the given id or name.
+
+Notice that passing a name on the url and a different name on the payload is not allowed.
+
+#### Response
+
+```
+HTTP 201 Created or HTTP 200 OK
+```
+
+See POST and PATCH responses.
+
+---
+
```

Comment:
As usual, those new endpoints need to be added to the YAML definition of the sidebar links at https://github.com/Kong/getkong.org/blob/master/app/_data/docs_nav_0.13.x.yml